fades/parsing.py

```diff
@@ -232,12 +233,33 @@ def parse_manual(dependencies):
     return _parse_requirement(dependencies)
 
 
+def _read_lines(filepath):
+    """Read a req file to a list to support nested requirement files."""
+    with open(filepath, 'rt', encoding='utf8') as fh:
+        for line in fh:
+            line = line.strip()
+            if line.startswith("-r"):
+                logger.debug(
+                    "Reading deps from nested requirement file: %s", line)
+                try:
+                    nested_filename = line.split()[1]
+                except IndexError:
+                    logger.warning("Invalid format to indicate a nested "
+                                   "requirements file: '%s'", line)
```

Comment:
This is a warning situation, probable the user is doing something differently than was intended, so better to show a repr() of the line (use ``%r``)